.kuzzlerc.sample

```diff
@@ -180,13 +180,34 @@
         }
       }
     },
-    // [standard]
-    // Permissions used when creating an administrator user, either using the
-    // CLI or the Back Office.
+    // [initial]
```

Comment:
For the wording, maybe `bootstrap` is better ?

I added theses roles definition here to avoid writing theses bases roles in the internal index bootstrap. I agree that defining `admin` and `default` roles is not very useful, I can remove theses roles from the kuzzlerc and just let the `anonymous` role. What do you think?